.github/workflows/lint-and-test-workflow.yml

```diff
@@ -1,8 +1,11 @@
 name: Lint and test
 
-# By default, runs when a pull request is opened, synchronized, or reopened.
-on: pull_request
-
+# By default, runs when pushed to master or a pull request is opened, synchronized, or reopened.
```

Comment:
I think I originally made this only run on PRs because it felt a duplicative to have a PR do these tests, pass, merge it, and then run the same tests again. Do you think we need it?

Reply:
I only included it here because the deploy script also ran the tests, but im not running test on cloud build (i dont mind running tests on cloud build, but i will also need to up the runtime on cloud build).